calibrate.py

```diff
@@ -129,7 +127,7 @@ def parse_args():
     parser.add_argument("-ih", "--invertHorizontal", dest="invert_h", default=False, action="store_true",
                         help="Invert horizontal axis of the camera for the display")
     parser.add_argument("-ep", "--maxEpiploarError", default="0.7", type=float, required=False,
-                         help="Sets the maximum epiploar allowed with rectification")
+                         help="Sets the maximum epiploar allowed with rectification. Default: %(default)s")
```

Comment:
what is this default set to.?

Reply:
It is set to 0.7. Should I set it lower?